### PR TITLE
Fix typo in perl script

### PR DESCRIPTION
**Pull request by emacser0**
minium -> minimum
maxium -> maximum

##### Checklist

- [ ] documentation is added or updated
- [ ] tests are added or updated

### CODE REVIEW

File: util/perl/OpenSSL/paramnames.pm [hunk 1]
```diff
@@ -220,8 +220,8 @@ my %params = (
 # DRBG call back parameters
     'DRBG_PARAM_ENTROPY_REQUIRED' =>        "entropy_required",
     'DRBG_PARAM_PREDICTION_RESISTANCE' =>   "prediction_resistance",
-    'DRBG_PARAM_MIN_LENGTH' =>              "minium_length",
-    'DRBG_PARAM_MAX_LENGTH' =>              "maxium_length",
+    'DRBG_PARAM_MIN_LENGTH' =>              "minimum_length",
+    'DRBG_PARAM_MAX_LENGTH' =>              "maximum_length",
```
Comment (tom-cosgrove-arm) [inline util/perl/OpenSSL/paramnames.pm:224]: Hmm, is fixing these going to cause workflows to fail for people who are currently working round this with the incorrect spellings?

Comment (paulidale) [reply util/perl/OpenSSL/paramnames.pm:224]: Very likely.
Compatibility rules supreme.
Unfortunately.

Comment (emacser0) [reply util/perl/OpenSSL/paramnames.pm:224]: I think also the problems you are worried about could possibly happen.

**Comment by emacser0** [comment]
Closed, It could cause compatibility issue.

File: util/perl/OpenSSL/paramnames.pm (hunk 1 above)
Comment (tom-cosgrove-arm) [reply util/perl/OpenSSL/paramnames.pm:224]: Is there some way to add the correct spellings in, with the current incorrect ones for backwards compatibility? Although I don't think that would count as trivial...

**Pull request closed without merging**